Conrollers/postConroller.js

```diff
@@ -10,5 +10,14 @@ const postArticle = (req, res, next) => {
       next(err);
     });
 };
-
-module.exports = { postArticle };
+function patcharticle_id(req, res, next) {
+  patcharticle_idMdl(req)
+ .then((data) => {
+  res.status(201).send(data);
```

Comment:
this will also be a status 200 as apart from POST we use 200 as a general success status